js_machine_front/src/App.tsx

```diff
@@ -1,26 +1,15 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
+import React from 'react';
+import './App.css';
+import { Routes } from 'components/Routes';
+import { NavBar } from 'components/NavBar';
+
+const App: React.FC = () => {
+  return (
+    <div className="App">
+      <Routes />

```

Comment:
please do not create separate components for routes, they all must be in App.tsx

Reply:
fix